src/gluonnlp/data/transforms.py

```diff
@@ -731,19 +730,18 @@ def __call__(self, sample):
 
         Parameters
         ----------
-        sample: list(str)
-            The sentence to detokenize
+        sample: str
+            The string to tokenize, the encoding must be Unicode.
```

Comment:
Only Python byte strings have an encoding, unicode strings do not. So the current documentation is confusing.

Reply:
Python2 has `str` and `unicode` two types of strings, how do I describe python2 users need to use `unicode` type string?

Reply:
In Python2 `str` is byte string and `unicode` is a unicode string. In Python3  `str` is unicode string and `bytes` a byte string. Just replacing "The string to tokenize, the encoding must be Unicode." with "The string to tokenize. Must be unicode." and "sample: str" with "sample: str (unicode for Python 2)" should be fine I guess. Thanks!